crates/swc_ecma_minifier/src/compress/optimize/props.rs

```diff
@@ -248,7 +248,10 @@ impl Optimizer<'_> {
             return;
         }
 
-        if self.ctx.is_update_arg || self.ctx.is_callee || self.ctx.is_exact_lhs_of_assign {
+        if self.ctx.bit_ctx.contains(BitCtx::IsUpdateArg)
+            || self.ctx.bit_ctx.contains(BitCtx::IsCallee)
+            || self.ctx.bit_ctx.contains(BitCtx::IsExactLhsOfAssign)
+        {
```

Comment:
```suggestion
        if self.ctx.bit_ctx.contains(BitCtx::IsUpdateArg | BitCtx::IsCallee | BitCtx::IsExactLhsOfAssign)
        {
```

Not sure if rustc will be able to optimize this, but this would be better in worst case

Reply:
```suggestion
        if self.ctx.bit_ctx.intersects(BitCtx::IsUpdateArg | BitCtx::IsCallee | BitCtx::IsExactLhsOfAssign)
        {
```

Replacing `contains` with `intersects` better matches the original implementation.

Reply:
Oh you are right. Sorry I was too sleepy at the moment because of the zet lag.